### PR TITLE
Za - update UI

### PR DESCRIPTION
**Pull request by Akintobi41**
# Why this PR?
to update sections of the UI to make it clean and presentable

# What does this PR do?
updates UI across the application.

# How to test your work?
navigate to the feeds/announcements/staff/students , click on either of the buttons for the filter or add category and the styles should be properly formatted. the media page also reflects some changes to the images.

# Pages to see your changes?
/feeds
/announcements
/staff
/students

### CODE REVIEW

File: src/components/pages/Announcements/Filter/index.tsx [hunk 1]
```diff
@@ -97,7 +97,7 @@ const FilterAnnouncement = ({ closeModal, action }: TFilterAnnouncement) => {
               </CustomField>
             </div>
           </div>
-          <div className="mt-4">
+          <div className={`mt-4 ${!fromDate ? 'pb-4' : ''}`}>
```
Comment (ibraheemosule) [inline src/components/pages/Announcements/Filter/index.tsx:100]:
```suggestion
          <div className="my-4">
```
is there a reason why we can't use this?

File: src/components/pages/Announcements/Filter/index.tsx [hunk 2]
```diff
@@ -119,7 +119,7 @@ const FilterAnnouncement = ({ closeModal, action }: TFilterAnnouncement) => {
             </div>
           </div>
           {fromDate && (
-            <div className="mt-4">
+            <div className={`mt-4 ${fromDate ? 'pb-4' : ''}`}>
```
Comment (ibraheemosule) [inline src/components/pages/Announcements/Filter/index.tsx:122]:
```suggestion
          <div className="my-4">
```
is there a reason why we can't use this?

File: src/components/pages/Media/index.tsx [hunk 3]
```diff
@@ -10,12 +10,12 @@ import ListOptions from '~components/reusables/ListOptions';
...
-const c =
-  'https://res.cloudinary.com/ibraheemsulay/image/upload/v1661931985/hippo%20labs%20task/surgeons2_tawgyq.jpg';
-const d =
-  'https://res.cloudinary.com/ibraheemsulay/image/upload/v1645458413/beatchain_nszwlv.png';
+// const b =
+//   'https://res.cloudinary.com/ibraheemsulay/image/upload/v1645458430/bincom_xyrzzj.png';
+// const c =
+//   'https://res.cloudinary.com/ibraheemsulay/image/upload/v1661931985/hippo%20labs%20task/surgeons2_tawgyq.jpg';
+// const d =
+//   'https://res.cloudinary.com/ibraheemsulay/image/upload/v1645458413/beatchain_nszwlv.png';
```
[the remark this answers is not in the document]
Comment (Akintobi41) [reply src/components/pages/Media/index.tsx:18]: No

File: src/components/pages/Announcements/Filter/index.tsx (hunk 1 above)
Comment (Akintobi41) [reply src/components/pages/Announcements/Filter/index.tsx:100]: I have adjusted it

File: src/components/pages/Announcements/Filter/index.tsx (hunk 2 above)
Comment (Akintobi41) [reply src/components/pages/Announcements/Filter/index.tsx:122]: I have adjusted it

**Review by ibraheemosule — APPROVED** [review]

**Pull request merged**